1-Draft/RFCNNN-WindowsPS-Compatibility-Utilities.md

```diff
@@ -0,0 +1,106 @@
+---
+RFC:
+Author: Andrew Menagarishvili
+Status: Draft
+SupercededBy:
+Version: 0.1
+Area: Language
+Comments Due: 10/10/2019
+Plan to implement: Yes
+---
+
+# Windows PowerShell Compatibility Utilities

```

Comment:
I think the title should be more user focused like: Importing Windows PowerShell Modules in PowerShell 7

Reply:
renamed.